ACMILabs/acmi-api
# #82 Add Constellations endpoint

Resolves #82

Let's add `/constellations/` to the Public API.

### Acceptance Criteria
- [x] Add `/constellations` to the API

### Relevant design files
* None

### Testing instructions
1. Start the development environment: `make up`
1. Note Constellations are on the home page: http://localhost:8081/
1. Visit the Constellations endpoint: http://localhost:8081/constellations/
1. Visit an individual Constellation: http://localhost:8081/constellations/1/
1. Update your search index: (see below)
1. Search Constellations: http://localhost:8081/search/?query=pen&resource=constellations

```python
$ docker exec -it api python

from app.api import Search
search = Search()
search.update_index('constellations')
```
